### ISSUE TITLE
Vimeo90K EDVR training not converging

### ISSUE THREAD

**Issue by PedroFO** [xinntao/BasicSR#357]
Trying to train a new model based on Vimeo90K dataset to achieve a similar performance to the pretrained model.

ISSUE: The training never converge.

Opt file used below:

        name: 001_EDVR_L_x4_SR_Vimeo90k_whatever
        model_type: EDVRModel
        scale: 4
        num_gpu: 2  # set num_gpu: 0 for cpu mode
        manual_seed: 10
        
        datasets:
          train:
        
            name: Vimeo90k
            type: Vimeo90KDataset
            dataroot_gt: /VSR/workspace/datasets/vimeo90k/vimeo_septuplet/sequences/
            dataroot_lq: /VSR/workspace/datasets/vimeo90k/vimeo_septuplet_matlabLRx4/sequences/
            meta_info_file: basicsr/data/meta_info/meta_info_Vimeo90K_test_slow_GT.txt
            io_backend:
              type: disk
        
            num_frame: 5
            gt_size: 256
            random_reverse: false
            use_flip: true
            use_rot: true
        
            use_shuffle: true
            num_worker_per_gpu: 3
            batch_size_per_gpu: 4
            dataset_enlarge_ratio: 200
        
          val:
            name: Vimeo90k
            type: VideoTestVimeo90KDataset
            dataroot_gt: /VSR/workspace/datasets/vimeo90k/vimeo_septuplet/sequences/
            dataroot_lq: /VSR/workspace/datasets/vimeo90k/vimeo_septuplet_matlabLRx4/sequences/
            meta_info_file: basicsr/data/meta_info/meta_info_Vimeo90K_test_fast_GT.txt
            io_backend:
              type: disk
        
            cache_data: false
            num_frame: 5
            random_reverse: false
        
        network_g:
          type: EDVR
          num_in_ch: 3
          num_out_ch: 3
          num_feat: 128
          num_frame: 5
          deformable_groups: 8
          num_extract_block: 5
          num_reconstruct_block: 40
          center_frame_idx: ~
          hr_in: false
          with_predeblur: false
          with_tsa: false
        
        path:
          pretrain_network_g: ~
          strict_load_g: true
          resume_state: ~
        
        train:
          optim_g:
            type: Adam
            lr: !!float 4e-4
            weight_decay: 0
            betas: [0.9, 0.99]
        
          scheduler:
            type: CosineAnnealingRestartLR
            periods: [150000, 150000, 150000, 150000]
            restart_weights: [1, 0.5, 0.5, 0.5]
            eta_min: !!float 1e-7
        
            
          total_iter: 600000
          warmup_iter: -1 
          dcn_lr_mul: 1
        
          pixel_opt:
            type: CharbonnierLoss
            loss_weight: 1.0
            reduction: sum
        
        val:
          val_freq: !!float 5e3
          save_img: false
        
          metrics:
            psnr
              type: calculate_psnr
              crop_border: 0
              test_y_channel: false
        
        
        logger:
          print_freq: 100
          save_checkpoint_freq: !!float 5e3
          use_tb_logger: true
          wandb:
            project: ~
            resume_id: ~
        
        
        dist_params:
          backend: nccl
          port: 29500

**Comment by xinntao** [comment]
Training a large EDVR is unstable due to the difficulty of learning meaning offsets. 
You may try to decrease the lr: !!float 4e-4 to 2e-4
